packages/dtslint/test/dt-header.test.ts

```diff
@@ -28,16 +28,37 @@ ruleTester.run("dt-header", dtHeader, {
       `,
       errors: [
         {
-          column: 2,
+          column: 1,
```

Comment:
This PR updates the version of `parsimmon`, a dependency of `header-parser`, from 1.13 to 1.18. Version 1.17 of `parsimmon` changed reporting of error line/column, and so this test needs to be updated to reflect that.